core/src/test/scala/io/finch/internal/ToResponseSpec.scala

```diff
@@ -0,0 +1,24 @@
+package io.finch.internal
+
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.Buf
+import com.twitter.io.Charsets
+import com.twitter.util.Await
+import org.scalatest.prop.Checkers
+import org.scalatest.{FlatSpec, Matchers}
+
+class ToResponseSpec extends FlatSpec with Matchers with Checkers {
```

Comment:
If you extend `FinchSpec` instead, you will get an arbitrary instance for `Buf` so you can simplify the test to:

```scala
check { (chunks: List[Buf]) =>
  val in = AsyncStream.fromSeq(chunks))
  val out = AsyncStream.fromReader(ToResponse.Aux[AsyncStream[Buf], Text.Plain].apply(in, Charsets.Utf8).reader)
  Await.result(in.toSeq) === Await.result(out.toSeq)
}
```